browser/about_flags.cc

```diff
@@ -224,6 +225,9 @@ constexpr char kNativeBraveWalletName[] = "Enable Brave Wallet";
 constexpr char kNativeBraveWalletDescription[] =
     "Native cryptocurrency wallet support without the use of extensions";
 
+constexpr char kBraveNewsName[] = "Enable Brave News";
+constexpr char kBraveNewsDescription[] = "Brave news feed feature";
```

Comment:
nit: other features are more descriptive. I would reach out to Ryan McFadden for Copy

Reply:
Please use "Brave News is completely private and includes anonymized ads matched on your device."

Reply:
done